code/controllers/configuration.dm

```diff
@@ -710,9 +710,11 @@ GLOBAL_PROTECT(config_dir)
 							WRITE_FILE(GLOB.config_error_log, "Unknown game mode probability configuration definition: [prob_name].")
 					else
 						WRITE_FILE(GLOB.config_error_log, "Incorrect probability configuration definition: [prob_name]  [prob_value].")
-				if("adjust")
+				if("repeated_mode_adjust")
 					if(value)

```

Comment:
should probably .Cut() the list before this, in case the config entry is duped

Reply:
Sure.